controllers/teleport_master_controller/teleport_master_controller.cpp

```diff
@@ -0,0 +1,144 @@
+/*
+ * This file is part of the NUbots Codebase.
+ *
+ * The NUbots Codebase is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The NUbots Codebase is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2021 NUbots <nubots@nubots.net>
+ */
+ 
+// Special thanks to Hamburg Bit-Bots for ideas for this controller
+// https://github.com/bit-bots/wolfgang_robot/blob/feature/recognition/wolfgang_webots_sim/src/wolfgang_webots_sim/webots_camera_controller.py
+
+#include <array>
+#include <fstream>
+#include <iostream>
+#include <math.h>
+#include <random>
+#include <webots/Supervisor.hpp>
+
+#include "yaml-cpp/yaml.h"
```

Comment:
System includes should use angle brackets
```suggestion
#include <yaml-cpp/yaml.h>
```